src/elements/content-sidebar/activity-feed/comment/Comment.scss

```diff
@@ -18,11 +18,11 @@
 
     .bcs-comment-confirm-delete,
     .bcs-comment-confirm-cancel {
-        margin: 0;
+        margin-left: 0;
```

Comment:
Side note: we really need a ButtonGroup component that does this automatically

Reply:
s it still be considered a group if the buttons are visually separated? If so, you're welcome to extend the [ButtonGroup component](http://opensource.box.com/box-ui-elements/#/Components?id=buttongroup) to support it.

Reply:
This isn't really a fix because the buttons aren't grouped. The design called for a narrow window, and the button margins pushed them to flow. A design without this outer constraint would allow us to remove this `margin` override entirely.

[`ButtonGroup`](https://opensource.box.com/box-ui-elements/#!/ButtonGroup) makes sense currently, and these two are associated with each other in the same way two words in a sentence are. A `FlexLayout` component of some sort might be a better direction to go in (sorta like the customize toolbar option in macOS). we could compose the layout with buttons together that way, and repurpose the layout if desired. The first consumer of that would be our other modal components, header bars, etc.

Reply:
Good point, it's more like this: https://bulma.io/documentation/elements/button/#list-of-buttons

(also Bootstrap/Zurb seem to have removed default horizontal margins from their buttons, so they are left-aligned by default)